<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPSolverEdit</class>
 <widget class="QMainWindow" name="LPSolverEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LP Solver Edit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Max time (sec)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Equal lessons weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Equal subjects weight</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_time">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="equal_lesson_weight">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="equal_subject_weight">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>328</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
